lib/puppet/network/http/api/v3.rb

```diff
@@ -14,9 +14,13 @@ class Puppet::Network::HTTP::API::V3
     Environments.new(Puppet.lookup(:environments))
   end)
 
-  def self.routes
-    Puppet::Network::HTTP::Route.path(%r{/v3}).
+  def self.master_routes
+    Puppet::Network::HTTP::Route.path(%r{v3}).
         any.
         chain(ENVIRONMENTS, INDIRECTED)
   end
+
+  def self.ca_routes
+    Puppet::Network::HTTP::Route.path(%r{v1}).any.chain(INDIRECTED)
```

Comment:
That's a good question. I think it's sort of weird right now because both the CA prefix at v1 and the master prefix at v3 end up in the same place - `IndirectedRoutes`. Other than here, the only place I can think that makes sense to define these would be directly in `http/api.rb`.

If we were bumping the CA routes to v2, then presumably it would be because we were separating them out to no longer use `IndirectedRoutes`, in which case it seems like it would make more sense to define them in their own file.